browser/components/CodeEditor.js

```diff
@@ -153,8 +153,69 @@ export default class CodeEditor extends React.Component {
         'Cmd-T': function (cm) {
           // Do nothing
         },
+        'Ctrl-I': function(cm) {
```

Comment:
For macOS, we have to implement this to `Cmd-I` and `Cmd-B`.
So, we have to extract the handler and make it work depending on OS.
Like the below:
```js
'Cmd-I': function (cm) {
  if (global.process.platform !== 'darwin') { return }
  applyItalic(cm)
},
'Ctrl-I': function (cm) {
  if (global.process.platform === 'darwin') { return }
  applyItalic(cm)
}
```